CMakeLists.txt

```diff
@@ -17,6 +17,10 @@ set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
+# This warning has a false positive. See
```

Comment:
It'd be nice to have an easy / automated way to know when this can be removed again.
But it's not worth putting additional effort into it.

Reply:
CMake has a feature that allows the output of a program to be used as the value of a variable.

I wrote a `bin/spurious-warning` script that uses `g++` and `grep` to demonstrate whether the spurious warning is produced. But, I had trouble coming up with a minimum reproducing program. In particular, the snippet in the linked bugzilla report didn't trigger the warning in GCC 12 (which is what recent Alpines use).

You're right, it's not worth putting additional effort into it.